### ISSUE TITLE
kubectl reference does not contain -k flag

### ISSUE THREAD

**Issue by kbhawkey** [kubernetes/website#13961]
**This is a Bug Report**

**Problem:**
- kubectl reference does not describe new -k flag for `apply` command.
- Was the kubectl reference updated for the 1.14 release?

**Proposed Solution:**
Update reference docs in k8s/k8s or just re-generate the kubectl reference docs?

See: 
https://kubectl.docs.kubernetes.io/pages/app_management/apply.html
https://kubernetes.io/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/#apply-and-verify

**Page to Update:**
https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply

**Kubernetes Version:**
1.14

**Comment by kbhawkey** [comment]
@jimangel, can you confirm that the kubectl reference was updated for the 1.14 docs release?  Trying to track down reference changes.

**Comment by kbhawkey** [comment]
Note: I am investigating the generation and instructions to generate.

**Comment by jimangel** [comment]
Hey @kbhawkey, sorry I'm late to the party; a great spot to track the summary of 1.14 reference docs is in this open issue: https://github.com/kubernetes/website/issues/13396 which links to the PRs I tested and the PRs merged that @tengqm did: https://github.com/kubernetes/website/pull/13444 - I see the kubectl bits in there, I wonder if it was a timing issue too...

One thing to consider for `kubectl` in particular is where https://kubectl.docs.kubernetes.io/ and https://kubernetes.io/ intersect. Should we focus more on empowering that sub doc project, or as @tengqm said during our meeting, focus on coordinating with the upstream providers to ensure we're offering a solid service by generating these documents.

It definitely is a pain point in the current release process.

/cc @MAKOSCAFEE for awareness

**Comment by kbhawkey** [comment]
Hi, @jimangel. I'll look at the open issues for 1.14. I have not been following along.
This issue was reported as the ref did not seem to contain the latest 1.14 changes for apply. I decided to try and re-generate the doc; the gen instructions have gone astray. I documented the changes I made. Totally agree that there are known gen issues/pain points. I would support having the reference generated as part of the tool. The publishing process and the ref UI could use some change. Though, the reference still seems valuable as part of the docs set.
